cores/esp8266/Schedule.cpp

```diff
@@ -249,3 +249,15 @@ void run_scheduled_recurrent_functions()
 
     fence = false;
 }
+
+
+bool yieldUntil (std::function<bool()> until, uint32_t maxMs) {
```

Comment:
std::function is not necessary for this use-case as the function obj is destroyed right after (it is relatively cheap though, but it's still a type pun hidden in the .cpp implementation)
plus this should probably be inside of esp8266 ns?

should this be a schedule.h template instead?
```cpp
namespace esp8266 {
namespace utils {

template <typename T>
bool yieldUntil(T&& untilCallback, unsigned long uint32_t) {
...
}

} // namespace utils
} // namespace esp8266
```

Reply:
`esp8266::utils::yieldUntil()` is moved in `cores/esp8266/yieldUntil.h`